packages/core/mock.ts

```diff
@@ -1,15 +1,40 @@
 import type { CompanyWrite, JobListingWrite, UserWrite } from "@dotkomonline/types"
+import type { GetUsers200ResponseOneOfInner } from "auth0"
 import { addWeeks, addYears } from "date-fns"
+import { ulid } from "ulid"
+
+export const getAuth0UserMock = (write?: Partial<UserWrite>): GetUsers200ResponseOneOfInner =>
+  ({
+    user_id: write?.auth0Id ?? "auth0|test",
+    email: write?.email ?? "fakeemail@gmai.com",
+    given_name: write?.givenName ?? "Ola",
```

Comment:
Should <code>given_name</code> include the middle name as we don't have a separate field for it. This way it's possible to update the middle-name as well.
 ```suggestion
    given_name: write?.givenName ?? "Ola Mellomnavn",
```

Reply:
How we have it now we have three fields related to the name. 
- given_name = first name
- family_name = last name
- name = full name including possible middle names. 

We will probably just change name to be given+family every time one of them changes? 

IMO we can just let people add their middle names to their given_name if they want maybe?

Reply:
I didn't know middle_name was a standard claim. So in this case yeah I agree let's add middle name

Reply:
Well the standard claims are all optional (except for sub), so that's totally fine and compliant.

Reply:
Okay yeah true. Middle name is now stored as it's own field in `user_metadata`! 👍